visallo/visallo
# Create a new base class for OntologyRepository tests

This will make it easier to test that all implementations of OntologyRepository are operating as expected. As part of implementing this, I updated the InMemoryOntologyRepository to fix a few ways that it was behaving differently than the VertexiumOntologyRepository.

- [x] mwizeman joeybrk372 rygim jharwig EvanOxfeld

Testing Instructions:
Run `mvn clean test` and ensure that all modules succeed.

CHANGELOG
Added: A new test helper named `OntologyRepositoryTestBase` that can be used to test implementations of `OntologyRepository`
